payments/schema/types.py

```diff
@@ -55,21 +56,39 @@ def resolve_default_product(self, info):
         return self.products.filter(harbor__isnull=True).first()
 
 
-class BerthProductNode(DjangoObjectType):
+class AbstractPlaceProductNode:
     price_value = graphene.Decimal(required=True)
     price_unit = PriceUnitsEnum(
         required=True, description="`Fixed to PriceUnit.AMOUNT`"
     )
     tax_percentage = PlaceProductTaxEnum(required=True)
+
+
+class BerthProductNode(DjangoObjectType, AbstractPlaceProductNode):
     price_group = graphene.Field("payments.schema.BerthPriceGroupNode", required=True)
     harbor = graphene.Field("resources.schema.HarborNode")
 
     class Meta:
         model = BerthProduct
         interfaces = (graphene.relay.Node,)
+        view_permissions = (BerthProduct,)
```

Comment:
what does this do and where does it come from? couldn't find mentions of it in `graphene-django`

Reply:
Whoops 😅 that was left from some tests I was doing. Removed now.

Reply:
Whoops 😅 that was left from some tests I was doing. Removed now.